vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/Tabs.java

```diff
@@ -91,7 +99,8 @@ public Tabs(Tab... tabs) {
      * with {@link Orientation#HORIZONTAL HORIZONTAL} orientation.
      *
      * @param autoselect
-     *            {@code true} to autoselect tab, {@code false} to not
+     *            {@code true} to automatically select the first added tab,
+     *            {@code false} to not
```

Comment:
`to not`? 🤔 Should there be something after that?

Reply:
done